specification.md

```diff
@@ -1795,12 +1864,15 @@ and respond with an offer of connection settings for the Agent.
 ```protobuf
 message ConnectionSettingsRequest {
     OpAMPConnectionSettingsRequest opamp = 1;
+    SettingsRequest settings_request = 2;
 }
 ```
 
 The `opamp` field is set to indicate a request for OpAMP connection settings.
-If this field is unset then the ConnectionSettingsRequest message is empty and is
-not actionable for the Server.
+The `settings_request` field is set to indicate a request for all offered
```

Comment:
"for all" or specifically "for TLSConnectionSettings"? What does "all" mean in this context?

Reply:
All settings for OpAMP  connection (this may include the hosts, and things like the heartbeat)